src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -7,6 +7,7 @@
     <SkipBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</SkipBuild>
     <PlatformPackageType>ToolPack</PlatformPackageType>
     <SharedFrameworkName>$(SharedFrameworkName).Crossgen2</SharedFrameworkName>
```

Comment:
I think we also need to add the PgoSuffix back:

```suggestion
    <SharedFrameworkName>$(SharedFrameworkName).Crossgen2</SharedFrameworkName>
    <PgoSuffix Condition="'$(PgoInstrument)' != ''">.PGO</PgoSuffix>
```

Reply:
We don't need to add the PGO suffix back because we don't produce this package in the PGO builds.

Reply:
I think what Alexander was saying is that you are reading from the `PgoSuffix` property again with this change but `$(PgoSuffix)` is nowhere defined.

Reply:
https://github.com/dotnet/runtime/pull/99246